### PR TITLE
use eslint `--cache`

### PR DESCRIPTION
**Pull request by o-az**

### CODE REVIEW

**Comment by pi0** [comment]
Nice! It seems renovate is also using this option (https://github.com/renovatebot/renovate/pull/16151) seems safe to try!

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "build": "unbuild",
     "dev": "vitest dev",
-    "lint": "eslint --ext .ts,.js,.mjs,.cjs .",
+    "lint": "eslint --ext .ts,.js,.mjs,.cjs . --cache",
```
Comment (pi0) [inline package.json:25]:
```suggestion
    "lint": "eslint --cache --ext .ts,.js,.mjs,.cjs .",
```

**Comment by o-az** [comment]
@pi0 I'm happy to do this on the other repos in unjs that use eslint. Any specific repos that I shouldn't make this change/pr for?

**Comment by pi0** [comment]
Let's gradually opt-in to cache option. I will start using it for new packages. Do you mind to add integration for nitro first?

**Comment by o-az** [comment]
Sounds good to me 👍

**Pull request merged**